docs/docs/examples/guides/05-jtc-guide.md

```diff
@@ -0,0 +1,587 @@
+self._trajectory.header.stamp = rospy.Time.now()
+self._trajectory.joint_names = ['joint_0', 'joint_1', 'joint_2', 'joint_3', 'joint_4', 'joint_5']
+
+p1 = JointTrajectoryPoint()
+p1.positions = [0.0, 0.5, 1.0, -0.5, 0.0, 0.2]
+p1.velocities = [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
+p1.time_from_start = rospy.Duration(2.0)
+
+p2 = JointTrajectoryPoint()
+p2.positions = [0.2, 0.4, 0.8, -0.2, 0.1, 0.3]
+p2.velocities = [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
+p2.time_from_start = rospy.Duration(4.0)
+
+self._trajectory.points = [p1, p2]
+
+self.publish_output("trajectory")
+```
+
+</TabItem>
+<TabItem value="c++" label="C++">
+
+```cpp title="include/custom_component_package/CppComponent.hpp"
+#pragma once
+
+#include <trajectory_msgs/msg/joint_trajectory.hpp>
+#include <trajectory_msgs/msg/joint_trajectory_point.hpp>
+
+// ... 
+std::shared_ptr<trajectory_msgs::msg::JointTrajectory> trajectory_;
+```
+
+```cpp title="src/CppComponent.cpp"
+#include "custom_component_package/CppComponent.hpp"
+
+// Typically in your constructor
+this->trajectory_ = std::make_shared<trajectory_msgs::msg::JointTrajectory>();
+this->add_output("trajectory", this->trajectory_, "", false, false);
+
+// Typically in a conditional block that will write the output upon the success of some criteria
+this->trajectory_->header.stamp = this->get_clock()->now();
+this->trajectory_->joint_names = {"joint_0", "joint_1", "joint_2", "joint_3", "joint_4", "joint_5"};
+this->trajectory_->points.resize(2);
+
+this->trajectory_->points[0].positions = {0.0, 0.5, 1.0, -0.5, 0.0, 0.2};
+this->trajectory_->points[0].velocities = {0.0, 0.0, 0.0, 0.0, 0.0, 0.0};
+this->trajectory_->points[0].time_from_start = rclcpp::Duration::from_seconds(2);
+
+this->trajectory_->points[1].positions = {0.2, 0.4, 0.8, -0.2, 0.1, 0.3};
+this->trajectory_->points[1].velocities = {0.0, 0.0, 0.0, 0.0, 0.0, 0.0};
+this->trajectory_->points[1].time_from_start = rclcpp::Duration::from_seconds(4);
+
+this->publish_output("trajectory");
+```
+</TabItem>
+</Tabs>
+
+:::tip
+The joint names need to correspond to the joint names from your robot description (*i.e.*, URDF). If you are **not**
+using the `Generic six-axis robot arm`, you will need to adjust these names.
+
+The easiest way to do so if you do not already know them is to head to the `Hardware` menu of your AICA Launcher and
+read through your URDF.
+:::
+
+#### Using JTC's `Set Trajectory` service
+
+The service currently accepts 5 variables, namely:
+
+- `times_from_start`: a list of timestamps (in seconds) measured from the start, indicating when JTC should reach each
+frame.
+- `frames`: names of the Cartesian frames to reach with the robot end effector 
+- `joint_positions`: names of joint positions that the robot should achieve
+- `blending_factors`: factors in [0.0, 1.0] indicating the amount of curving allowed between 2 consecutive waypoints.
+The first and last segment are currently not considered in blending, therefore, the vector's size needs to be equal to
```

Comment:
Do we already have an issue to improve that?

Reply:
Yes, we would be allowing blending for the first waypoint (never the last)